app/js/controllers/gesture.js

```diff
@@ -0,0 +1,79 @@
+/*global Controller*/
+
+export default class GestureController extends Controller {
+  constructor(options) {
+    super(options);
+
+    var firstTouchStartEvt = null;
+    var isForwarding = false;
+
+    window.addEventListener('touchstart', (evt) => {
+      if (evt.touches.length === 1) {
+        firstTouchStartEvt = evt;
+        return;
+      }
+
+      if (evt.touches.length !== 2) {
```

Comment:
Regardless, we'll be forwarding 'touchmove' events for a third touch that hasn't had a 'touchstart' sent for it. So we should either not send 'touchmove' events associated with the third touch, or forward all touches, as long as there are at at least two, through.